solidity/utils/test_deployment.js

```diff
@@ -354,6 +366,13 @@ const run = async () => {
 
     await execute(checkpointStore.methods.grantRole(ROLE_OWNER, liquidityProtection._address));
 
+    await execute(stakingRewardsStore.methods.grantRole(ROLE_OWNER, stakingRewards._address));
+    await execute(stakingRewardsStore.methods.grantRole(ROLE_MANAGER, account.address));
+    await execute(stakingRewards.methods.grantRole(ROLE_UPDATER, account.address));
```

Comment:
There is no need to set the `ROLE_MANAGER` or the `ROLE_UPDATER` roles during deployment

Reply:
fixed